static/scss/review.scss

```diff
@@ -0,0 +1,62 @@
+.review-card {
+
+  &.review-panel-right {
+    border-left: 1px solid black;
+  }
+
+  .status {
+    width: 75%;
+    margin: auto;
+  }
+
+  .section  {
+    margin: 20px 0px;
+    padding-bottom: 10px;
+  }
+
+  .header, .list-header {
+    height: 24px;
+    color: #000000;
+    font-family: Helvetica;
+    font-size: 20px;
+    font-weight: bold;
+    letter-spacing: 0.63px;
+    line-height: 24px;
+  }
+
+  .profile-header {
+    background-repeat: no-repeat;
+    background-image: url(/static/images/icon-user-v2.png);
+    height: 70px;
+    align-content: center;
+
+    div {
+      padding-left: 80px;
+    }
+  }
+
+  .profile {
+    margin-top: 43px;
+  }
+
+  .radio-status {
+    display: block;
+    width: 100%;
+    height: 45px;
+    line-height: 24px;
+    padding-bottom: 20px;
```

Comment:
In dev tools I removed all of these definitions besides `line-height`. I think it looks good and it avoids 2 issues. If you need extra spacing you can use `margin-bottom`

1. Weird element sizing (before/after)
<img width="306" alt="ss 2020-06-12 at 16 19 03 " src="https://user-images.githubusercontent.com/14932219/84543059-9fa44100-acc8-11ea-9995-e932de013449.png">
<img width="319" alt="ss 2020-06-12 at 16 18 21 " src="https://user-images.githubusercontent.com/14932219/84543221-eabe5400-acc8-11ea-989c-9dd6626d368c.png">

2. Weird text wrapping when the window narrows
<img width="587" alt="ss 2020-06-12 at 13 33 28 " src="https://user-images.githubusercontent.com/14932219/84543111-b8145b80-acc8-11ea-9c4c-c38483c1a916.png">